lib/matplotlib/tests/test_axes.py

```diff
@@ -22,6 +22,10 @@
 import warnings
 from matplotlib.cbook import IgnoredKeywordWarning
 
+# Note: Some test cases are run twice: once normally and once with labeled data
```

Comment:
I am inclined to let this slide for now.  I was happy that this added effective tests with out adding any new test images!